devops/sql/migrate-21.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE chatroach.surveys_metadata (
+	surveyid UUID NOT NULL REFERENCES chatroach.surveys(id) ON DELETE CASCADE,
+	off_date TIMESTAMPTZ NOT NULL,
+	UNIQUE(surveyid)
```

Comment:
If it's unique and not null, it might be more expresive to just call it a primary key ?

Reply:
I would call it `survey_metadata` --  "data" is already plural, and it's not metadata about all the surveys, each row just refers to one survey!

Reply:
Sounds good.